apps/comps/e2e/tests/privy-auth.test.ts

```diff
@@ -132,234 +117,4 @@ describe("Privy Authentication", () => {
       rpcClient.user.linkWallet({ walletAddress: newWalletAddress }),
     ).rejects.toThrow(/Wallet not linked to user/);
   });
-
-  // Our middleware calls the `verifyPrivyIdentityTokenAndUpdateUser` function, which is used to
-  // backfill users to Privy. This function is used to handle the following cases:
-  // 1. Brand new users, which will have Privy-related information included from the get-go
-  // 2. Users who have not logged in with Privy, which includes:
-  //    - Users with wallet + email (i.e., they completed their legacy Loops signup)
-  //    - Users with wallet but no email (i.e., they never verified their email, pre-Privy)
-  describe("Backfilling users with Privy data", () => {
```

Comment:
we no longer "backfill" users like this, so these test all fail (as expected), and refacotring them would be ultimately be duplicative with tests in `user.test.ts`.